db/migrations/001_initial_schema.sql

```diff
@@ -0,0 +1,123 @@
+-- Initial PostgreSQL schema for queue system with bot isolation
+-- This migration sets up pgboss, bot isolation, and RLS policies
+
+-- Enable pgboss extension for queue functionality
+CREATE EXTENSION IF NOT EXISTS pgcrypto;
+
+-- Create bot table for multi-bot credential support
+CREATE TABLE bots (
+    id SERIAL PRIMARY KEY,
+    bot_id VARCHAR(100) NOT NULL UNIQUE, -- Platform bot ID (e.g., Slack bot ID)
+    platform VARCHAR(50) NOT NULL, -- slack, discord, teams, etc.
+    name VARCHAR(100) NOT NULL,
+    token_hash VARCHAR(255), -- Hashed bot token for verification
+    created_at TIMESTAMP DEFAULT NOW(),
+    updated_at TIMESTAMP DEFAULT NOW(),
+    is_active BOOLEAN DEFAULT TRUE,
+    UNIQUE(platform, bot_id)
+);
+
+-- Create users table with bot association
+CREATE TABLE chat_users (
+    id SERIAL PRIMARY KEY,
+    bot_id INTEGER REFERENCES bots(id) ON DELETE CASCADE,
+    platform_user_id VARCHAR(100) NOT NULL,
+    platform VARCHAR(50) NOT NULL,
+    github_username VARCHAR(100) NOT NULL,
```

Comment:
remove this here, instead add a user_configs table where we have environment_variables with hstore type and the user-id. The envs value will be used when the worker is started from the orchestrator. The user can change this value from its postgresql user from the worker.